docs/cli.md

```diff
@@ -85,36 +91,45 @@ www.vegeta.com
 www.gohan.com
 ```
 
+If you omit `-g`, it will only provide a list of invalid URLs. This might be
+useful if you want to valid quickly a list of URLs.
+
+
 The benchmark flag can be used to output the time it takes to process piped input:
 
 ```bash
-cat wikipedia_100k.txt | ./buildbench/tools/adaparse -b
+cat wikipedia_100k.txt | adaparse -b
 ```
 
 ```bash
-(---snip---)
-file:///opt 
-file:///Users 
-file:///Users/lemire 
-file:///Users/lemire/tmp 
-file:///Users/lemire/tmp/linuxdump 
-file:///Users/lemire/tmp/linuxdump/linuxfiles.txt 
-file:///.dockerenv 
-read 10124906 bytes in 3071328453 ns using 169312 lines
-0.003296588481153891 GB/s
+Invalid URL:1968:_Die_Kinder_der_Diktatur
```

Comment:
Should we add a space after `:`?

```suggestion
Invalid URL: 1968:_Die_Kinder_der_Diktatur
```

Reply:
That's what the code dumps now. Ok. Let us add a space.

Reply:
Done.